routes/movies.js

```diff
@@ -17,7 +17,7 @@ router.get("/", movieController.listMovies)
 router.get("/Home", movieController.Home)
```

Comment:
El home siempre deberia de estar en la raiz es mejor para el usuario así ya no tiene que poner /home para ver la pagina de inicio 😉 
```suggestion 
router.get("/", movieController.Home);

```